scripts/zones/Upper_Jeuno/npcs/Chocobo.lua

```diff
@@ -120,35 +118,29 @@ function onEventFinish(player,csid,option)
 
     if (csid == 0x0039) then
         player:setVar("ChocobosWounds_Event", 2);
-        player:setVar("ChocobosWounds_Day",VanadielDayOfTheYear());
-        player:setVar("ChocobosWounds_Year",VanadielYear());
+        player:setVar("ChocobosWounds_Min",os.time() + 60);

```

Comment:
it returns seconds since the unix epoch

On Fri, Nov 6, 2015 at 7:42 AM, bendangelo <notifications@github.com> wrote:

> In scripts/zones/Upper_Jeuno/npcs/Chocobo.lua
> <https://github.com/DarkstarProject/darkstar/pull/2375#discussion_r44142485>
> :
>
> > @@ -120,35 +118,29 @@ function onEventFinish(player,csid,option)
> >
> >      if (csid == 0x0039) then
> >          player:setVar("ChocobosWounds_Event", 2);
> > -        player:setVar("ChocobosWounds_Day",VanadielDayOfTheYear());
> > -        player:setVar("ChocobosWounds_Year",VanadielYear());
> > +        player:setVar("ChocobosWounds_Min",os.time() + 60);
>
> Wait os.time() returns minutes? Game minutes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/2375/files#r44142485>.
>